CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 - [plugin] added support for `isTransient` of `TerminalOptions` and `ExternalTerminalOptions` VS Code API [#12055](https://github.com/eclipse-theia/theia/pull/12055) - Contributed on behalf of STMicroelectronics
 - [terminal] added support for preference `terminal.integrated.enablePersistentSessions` to allow disabling restoring terminals on reload [#12055](https://github.com/eclipse-theia/theia/pull/12055) - Contributed on behalf of STMicroelectronics
 
+<a name="breaking_changes_1.34.0">[Breaking Changes:](#breaking_changes_1.34.0)</a>
+
+- [plugin-ext] `DebugExtImpl#sessionDidCreate` has been replaced with `DebugExtImpl#sessionDidStart` to avoid prematurely firing a `didStart` event on `didCreate` [#11916](https://github.com/eclipse-theia/theia/issues/11916)
```

Comment:
`didStart` and `didCreate` seem like two different events which both may have their use cases. I wonder why you removed the `didCreate`?

Reply:
https://github.com/eclipse-theia/theia/pull/11984/commits/2b79c77b314dc500d541b83eafa192557d55845a